experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -2645,6 +2645,7 @@ function testTreeView(): ISharedTreeView {
 		schema,
 	});
 }
+
 /**
```

Comment:
Probably revert this.

Were you planning on changing this file to use the helpers that you copied into the utils file rather than having them duplicated?

Reply:
I was and then decided to do it as part of a larger refactor.